### DISCUSSION TITLE
查重的可能性探讨

### DISCUSSION THREAD

**Discussion by sikouhjw** [sikouhjw/gdutthesis#37] [Announcements]
已知
1. texlive 2019（含 2019）之后的 `ctex` 宏集不会有『查重乱码』问题；
2. 最终提交为 `.pdf` 格式；
3. 『[学位论文格式检测机器人](https://news.gdut.edu.cn/viewarticle.aspx?articleid=159914)』系统提交的文件格式为 `.docx`，但『系统会对论文格式存疑情况进行提示，提示不算错误，不计入差错率。』，因此不重要；

目前在跟研究生办沟通，测试查重。

**Comment by sikouhjw** [comment]
在跟研究生办沟通查重的时候，发现查重网站的规定『为准确反应参考文献引用情况，需正确识别参考文献，请在正文后标明“参考文献”字样，并按照（GB/T 7714-2015）《信息与文献 参考文献著录规则》规范参考文献条目格式』。

而 `撰写规范.docx` 规定了『参考文献格式参照国家标准“GB7714-2005 文后参考文献著录规则”及“CAJ-CD 中国学术期刊（光盘版）检索与评价数据规范”编写。』。

看来直接用 GB/T 7714-2015 是没问题的。

**Comment by sikouhjw** [comment]
查重结果已出，具体表现为：
1. 正常文字部分查重正确
2. 参考文献被正确的忽略
3. 页眉页脚被算入查重部分
4. 目录被算入查重部分
4. 学位论文独创性声明及学位论文版权使用授权声明被算入查重部分
5. 断行、分段识别有所欠缺

改进：可以考虑设置一个查重选项，不输出页眉页脚，以达到更好的查重效果。具体得看学校是否有规定。

总结：本模板在『万方检测』查重没有问题。

**Reply by SiriusFang2020** [reply]
请问你是怎么在万方查重里上传PDF版本的论文呢？似乎只能上传doc、docx、txt文件
![image](https://user-images.githubusercontent.com/59632817/158772989-d39d8ba9-615d-4efb-a083-d2d0a6123108.png)

**Reply by sikouhjw** [reply]
> 请问你是怎么在万方查重里上传PDF版本的论文呢？似乎只能上传doc、docx、txt文件

是否可以直接上传 pdf？如果不可以直接上传，压缩为 zip 格式的压缩包再上传？

**Reply by checkpoint-x** [reply]
有老哥用压缩包查重成功了的吗

**Reply by sikouhjw** [reply]
跟研究生办沟通后，学校/学院统一查重的系统，是可以直接用 pdf 查重的。

可以试试找客服咨询能否上传 pdf，如果不能，就考虑从 pdf 里复制文字到 txt 里，再上传。

主要查重贡献都是文本给的，其它的应该不会有太多的重复率，与最终结果应该不会差距太大。

**Reply by checkpoint-x** [reply]
好的，感谢大佬

**Reply by sikouhjw** [reply]
经反馈，万方不允许个体用 pdf 查重，可以考虑转成 Word 或者复制到 txt 里查重。

**Comment by greens007** [comment]
> 已知
> 
> 1. texlive 2019（含 2019）之后的 `ctex` 宏集不会有『查重乱码』问题；
> 2. 最终提交为 `.pdf` 格式；
> 3. 『[学位论文格式检测机器人](https://news.gdut.edu.cn/viewarticle.aspx?articleid=159914)』系统提交的文件格式为 `.docx`，但『系统会对论文格式存疑情况进行提示，提示不算错误，不计入差错率。』，因此不重要；
> 
> 目前在跟研究生办沟通，测试查重。

请问研究生办那边回复了吗

**Reply by sikouhjw** [reply]
> 请问研究生办那边回复了吗

请查看全部内容再回复。

**Reply by greens007** [reply]
看到了，现在很慌，犹豫要不要用latex，格式没有问题就好，多谢你们一直维护啊

**Reply by sikouhjw** [reply]
> 看到了，现在很慌，犹豫要不要用latex，格式没有问题就好，多谢你们一直维护啊

主要担心的点还是查重吧，如果担心的话可以用 Word[^1]。

[^1]: https://github.com/sikouhjw/gdutthesis-Word/

**Reply by greens007** [reply]
查重没问题的，我好几篇小文章，全部弄上去就好了

**Reply by SiriusFang2020** [reply]
> 查重没问题的，我好几篇小文章，全部弄上去就好了

能否具体说一下在万方上面查重怎么操作呢？没太看懂什么叫做“全部弄上去”呢？

**Reply by sikouhjw** [reply]
> 能否具体说一下在万方上面查重怎么操作呢？没太看懂什么叫做“全部弄上去”呢？

他的意思可能是“我有几篇文章，都是原创的，抄上去不怕‘查重率’”，而不是担心模板“能否查重”。

**Reply by SiriusFang2020** [reply]
哦哦好的，多谢🙏

**Reply by greens007** [reply]
现在万方查重不了pdf嘛，其他平台呢

**Reply by checkpoint-x** [reply]
学院推荐知网 万方 paperpass，paperpass是可以上传pdf的

**Reply by greens007** [reply]
> 学院推荐知网 万方 paperpass，paperpass是可以上传pdf的

多谢~
